File: app/api/urls.py
from django.urls import path
from .views import CityList, CityDetail, HomeData, EventDetails, EventList


urlpatterns = [
    path('cities/', CityList.as_view()),
    path('cities/<int:pk>/', CityDetail.as_view()),
    path('home_cities/', HomeData.as_view()),
    path('event_list/', EventList.as_view()),
    path('events/<int:pk>/', EventDetails.as_view()),
]

File: templates/static/js/helpers/urls.js
export const getEnvironmentURL = (path = "") => {
    return `${window.location.protocol}//${window.location.host }/${path}`;
}

export const Urls = {
    environment: getEnvironmentURL(),
    cities: getEnvironmentURL('api/cities/'),
    home_cities: getEnvironmentURL('api/home_cities/'),
    events: getEnvironmentURL('api/events/'),
    event_list: getEnvironmentURL('api/event_list/')
}